kbase-extension/static/kbase/js/widgets/function_output/KBaseSets/SetBrowser.css

```diff
@@ -0,0 +1,55 @@
+.SetBrowser .Table {
```

Comment:
Could you move all of the css files (including those that were in the other samples PR) into the `css` folder so that it'll be easy to convert them all to scss when that glorious day arrives when we may finally merge the `truss` branch into `develop`?